src/pages/Accounts/AccountSettings/AccountSettingsEditDisplayPropertiesScreen.tsx

```diff
@@ -139,6 +141,10 @@ const styles = StyleSheet.create( {
   listFooterSection: {
     // paddingHorizontal: 16,
     alignItems: 'flex-start',
+    shadowColor: Colors.shadowBlue,
```

Comment:
I agree with @AliMeer's comment [here](https://github.com/bithyve/hexa/pull/2501/files#r557604152) about making component styles for this. Is there a reason that those needed to [be removed](https://github.com/bithyve/hexa/pull/2501/files#diff-39b5f3993b48e91a7254859feb4003eee3fbb6abfac6dce063849e8e28e7e435L35)?

Also, why would we be applying buttons styles to the  `listFooterSection` here, instead of to the button?

Reply:
if we use that shadow styling on the button then it's messed-up on an android device. We need to use that to the parent component.